include/wx/fontutil.h

```diff
@@ -144,6 +144,9 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
     // store the fractional point size separately if it was initially specified
     // as we can't losslessly recover it from LOGFONT later.
     float        pointSize;
+    // Keep track of the ppi that is used to calculate the logfont height from
+    // the pointSize.
+    int          ppi;
```

Comment:
IIRC we tried to avoid storing the PPI in `wxNativeFontInfo` itself initially. It's strange that we managed to get everything working without doing this except for, apparently, wxGC stuff. What's so different about it that it requires doing this?